owasp-top10-2017-apps/a7/comment-killer/README.md

```diff
@@ -0,0 +1,58 @@
+# Comment-killer
+
+Comment-killer is a simple JS app that contains an example of multiple Cross-Site Scripting vulnerabilities and its main goal is to describe how a malicious user could exploit them on this purposefully vulnerable app.
+
+<img src="image/img1.png" alt="img1.png"/>
```

Comment:
Could you please invert the order of the image and the `Comment-killer is a simple JS app that...` text to better fit our template? 🙂

Reply:
![image](https://user-images.githubusercontent.com/8943477/95387201-c1a07f80-08c6-11eb-8c95-0e93e7ed57da.png)